### DISCUSSION TITLE
PyAutoGUI：利用自動控制滑鼠跟鍵盤

### DISCUSSION THREAD

**Discussion by LeoTsai999** [1101-GS4719-C/Python-Community-GS4719-C#2118] [優秀Python套件]
PyAutoGUI是一個能夠自動化控制電腦的滑鼠與鍵盤的函式庫。我的觀點是，對於喜歡玩天堂、楓之谷等類型遊戲的玩家來說，這個函式庫可以幫助你做出遊戲掛機程式，讓你只要電腦開著就能掛網刷道具，不需要人坐在電腦前面一直按，非常方便！
除此之外，我還有想到另一個應用是配合網路爬蟲，製作自動化的校內羽球場搶場機器人。

**Comment by gigitilto** [comment]
想請問一下，我使用這個函式庫寫一個自動連點器，但他的響應速度不如我的預期，我想了很久還是不太知道原因，是這個函式庫存在問題嗎？

**Comment by lai1008** [comment]
聽起來是一個很酷的函式庫，可以做校內羽球搶場地機器人挺猛的，因為學校的羽球場在整修，這個功能真不錯

**Comment by 113607514** [comment]
這個想法真不錯！使用PyAutoGUI來製作遊戲掛機程式，讓玩家在遊戲中輕鬆刷道具，確實能提升遊戲體驗。而結合網路爬蟲來搶校內羽球場的場地也很有創意，這樣不僅省時，還能提高搶場成功率！期待看到你實現這些計畫，或許可以分享一些具體的程式碼或經驗，讓更多人受益！

**Comment by Jing-Asteroid** [comment]
感謝老哥(姊)，最近在研究爬蟲的東西要用來下載大量資料剛好很需要。
BTW， selenium 這個套件也可以用來爬蟲喔，模擬人類按滑鼠之類的。

**Comment by 112121py** [comment]
感覺很酷欸，對於一些需要掛著還要偶爾動一下的遊戲還蠻實用的xd ，感覺對於其他課程需要用搶的學校來說，不知道可不可以實作得出來搶課的程式，雖然不知道這樣真的合法嗎xd
